src/main/java/ch/njol/skript/conditions/CondIsActivate.java

```diff
@@ -0,0 +1,56 @@
+package ch.njol.skript.conditions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.conditions.base.PropertyCondition;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import org.bukkit.block.Block;
+import org.bukkit.block.CreatureSpawner;
+import org.bukkit.block.data.type.TrialSpawner;
+import org.bukkit.block.data.type.TrialSpawner.State;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Is Activated")
+@Description({
+	"This condition checks whether one or more blocks are activated.",
+	"Only supports Spawners and Trial Spawners (if a player is close enough)."
```

Comment:
this already exists in its own condition, this had that functionality before

Reply:
Yeah I know there's a redstone powered condition, what I'm saying is that I think the "activated" condition property could confuse people because it applies to this and not those.

Reply:
i was thinking that it would be better to have a generic condition just in case other blocks can be "activated", i think there is some that i can add actually